packages/manager/modules/overthebox/src/overtheboxes.controller.js

```diff
@@ -0,0 +1,22 @@
+import { ListLayoutHelper } from '@ovh-ux/ng-ovh-telecom-universe-components';
+
+export default class OverTheBoxesCtrl extends ListLayoutHelper.ListLayoutCtrl {
+  /* @ngInject */
+  constructor($q, ouiDatagridService) {
```

Comment:
Both `$q` and `ouiDatagridService` are really needed?

I can't find any references.

Reply:
needed by ListLayoutCtrl but indeed it could be injected through its constructor, we'll have a look to improve